core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -811,6 +811,11 @@ public enum Property {
           + " that begin with 'table.file' can be used here. For example, to set the compression"
           + " of the sorted recovery files to snappy use 'tserver.wal.sort.file.compress.type=snappy'.",
       "2.1.0"),
+  TSERV_FAILED_BULK_COPY_THREADS("tserver.failed.bulk.threads", "2", PropertyType.COUNT,
+      "The number of threads used in the distributed work queue for copying failed bulk import RFiles.",
```

Comment:
"distributed work queue" here may be an implementation detail that does help the user in setting this value.